airflow/providers/google/BACKPORT_README.md

```diff
@@ -0,0 +1,26 @@
+<!--
+ Licensed to the Apache Software Foundation (ASF) under one
+ or more contributor license agreements.  See the NOTICE file
+ distributed with this work for additional information
+ regarding copyright ownership.  The ASF licenses this file
+ to you under the Apache License, Version 2.0 (the
+ "License"); you may not use this file except in compliance
+ with the License.  You may obtain a copy of the License at
+
+   http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing,
+ software distributed under the License is distributed on an
+ "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ KIND, either express or implied.  See the License for the
+ specific language governing permissions and limitations
+ under the License.
+ -->
+
+##  Backport Google Package limitations:
+
+* Kubernetes Pod operators are imported directly from airflow.contrib.
+* BigQuery operators are removed until it is refactored. This is going to happen
+  soon and we recommend to use the 1.10 BigQuery operators from the airflow.contrib
+  package.
+* GKEStartPodOperator is removed as it uses functionality only available in 2.0.
```

Comment:
The following operators have not been made available:
* All operators for BigQuery service. 
* GKEStartPodOperator. 

We recommend staying with the old versions yet. These operators will be released as soon as possible.

Reply:
Feel free to add it.